### ISSUE TITLE
libretime translation in french

### ISSUE THREAD

**Issue by marmotte31** [LibreTime/libretime#805]
Hello guys,
Thanks for this brilliant soft!!!
How can I contribute and translate in French?

**Comment by Robbt** [comment]
So we should have a french translation already but it might need some improvement.
It is located in airtime_mvc/locale/fr_FR/LC_MESSAGES/airtime.po
or https://github.com/LibreTime/libretime/blob/master/airtime_mvc/locale/fr_FR/LC_MESSAGES/airtime.po

And #301 tracks our efforts to create a better interface for translating LibreTime

**Comment by Robbt** [comment]
I'm going to close this because it is more of a request for information than an issue. Thanks for your interest.

**Comment by hairmare** [comment]
Salut @marmotte31!

Je suis désolé d'être en retard pour répondre à ton offre de traduire LibreTime en français. Je vais te répondrai en anglais parce que c'est plus facile pour moi et les autres.

Currently, we've been on planning to use [Zanata](https://zanata.com) as our main "non techie" translation interface.

Right now the (technical and organizational) integration of this platform is still underway and it looks like it will take some time (ie. alpha going rc phase) until we can finish setting everything up.

For most languages I guess it will make sense to set up a designated translation team with up to three dedicated team leads and possibly more translators. Would you be willing to take a leading role in the fr_FR translation effort? Do you have any other inputs we need to consider for fr-FR or i18n in general?

For now, progress on our i18n efforts are tracked in #301 with my current update on the state of i18n in https://github.com/LibreTime/libretime/pull/301#issuecomment-516129090.

I'm re-opening this issue open and tagging it as i18n as a tracker for setting up the fr_FR translation team and what is needed from a techie point of view.

Please let us us know if you would like to consider taking a leading role in the fr_FR translation effort so we can get you on-boarded.

Cheers,
Lucas

**Comment by marmotte31** [comment]
Hi @hairmare,
Ok let's do that.
FYI 30% of the po french file is already reviewed and translated on my PC with GTranslator.

Is there any interested folks to translate LibreTime to french?

Also, I was wondering if we can change the occurence AirTime with LibreTime in the translation?

**Comment by paddatrapper** [comment]
> Also, I was wondering if we can change the occurence AirTime with LibreTime in the translation?

Yes please, but also file bugs or fix the original text too so that other translations and the English source reflects the correct naming
